pages/crash_report_page.py

```diff
@@ -33,7 +33,7 @@ class Report(CrashStatsBasePage):
 
         _product_locator = (By.CSS_SELECTOR, 'td:nth-of-type(3)')
         _version_locator = (By.CSS_SELECTOR, 'td:nth-of-type(4)')
-        _report_date_link_locator = (By.CSS_SELECTOR, '#reports-list .report-date_processed > a')
+        _report_date_link_locator = (By.CSS_SELECTOR, '#reports-list tbody td:nth-of-type(1) > a')
```

Comment:
I'll take another look to see if I can improve this .. it's been a long tail working with the team to get better locators for our tests.